### PR TITLE
Fix naming of transformation methods

### PR DESCRIPTION
**Pull request by webrouse**
Part of: https://keboola.atlassian.net/browse/COM-1104

### CODE REVIEW

File: internal/pkg/mapper/mapper.go [hunk 1]
```diff
@@ -20,26 +20,12 @@ func New(state *model.State, logger *zap.SugaredLogger, fs filesystem.Fs, naming
 	return &Mapper{state: state, logger: logger, fs: fs, naming: naming}
 }
 
-func (m *Mapper) isTransformationConfig(object interface{}) (bool, error) {
```
Comment (webrouse) [inline internal/pkg/mapper/mapper.go:23]: Presunute nizsie, pod public metody.

File: internal/pkg/mapper/mapper.go [hunk 2]
```diff
@@ -20,26 +20,12 @@ func New(state *model.State, logger *zap.SugaredLogger, fs filesystem.Fs, naming
...
-	return component.IsTransformation(), nil
-}
-
 func (m *Mapper) BeforeSave(files *model.ObjectFiles) error {
 	// Save transformation
 	if ok, err := m.isTransformationConfig(files.Object); err != nil {
 		return err
 	} else if ok {
-		return transformation.GenerateBlockFiles(
+		return transformation.Save(
```
Comment (webrouse) [inline internal/pkg/mapper/mapper.go:28]: Premenovane.

File: internal/pkg/mapper/mapper.go [hunk 3]
```diff
@@ -52,9 +38,9 @@ func (m *Mapper) BeforeSave(files *model.ObjectFiles) error {
 }
 
 func (m *Mapper) AfterLoad(files *model.ObjectFiles) error {
-	// Map transformation
+	// Load transformation
 	if ok, err := m.isTransformationConfig(files.Object); ok {
-		return transformation.LoadBlocks(
+		return transformation.Load(
```
Comment (webrouse) [inline internal/pkg/mapper/mapper.go:43]: Premenovane.

**Review by kacurez — APPROVED** [review]
lgtm

**Pull request merged**